src/v2/orders.ts

```diff
@@ -0,0 +1,130 @@
+import { Client } from '../client'
+import { BaseError } from '../errors'
+import { UriHelper } from '../uri-helper'
+import { ThBaseHandler } from '../base'
+
+
+export interface OrdersOptions {
+    user?: string
+    base?: string
+}
+
+export interface OrdersResponse {
+    data: Order[]
+    metadata: Record<string, unknown>
+    next?: () => Promise<OrdersResponse>
+}
+
+export interface OrdersQuery {
+    limit?: number
+    uri?: string
+    query?: {
```

Comment:
The query object seems to be missing most of it. HEre is the link to the filters that are expected
https://unz.atlassian.net/wiki/spaces/TA/pages/1159004297/UFC+Order+Management+Tech+Spec#3.1.4-Orders-API

Reply:
Think now it is good